dev/py3_10_contextlib_pyi_to_contextlib2_pyi.patch

```diff
@@ -50,12 +50,27 @@
      _S = TypeVar("_S", bound=AsyncExitStack)
  
      _ExitCoroFunc = Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], Awaitable[bool]]
-@@ -112,7 +121,7 @@
+@@ -112,7 +121,8 @@
              __traceback: Optional[TracebackType],
          ) -> Awaitable[bool]: ...
  
--if sys.version_info >= (3, 7):
+-if sys.version_info >= (3, 10):
++
 +if True:
-     class nullcontext(AbstractContextManager[_T]):
+     class nullcontext(AbstractContextManager[_T], AbstractAsyncContextManager[_T]):
          enter_result: _T
          @overload
+@@ -123,13 +133,3 @@
+         def __exit__(self, *exctype: Any) -> None: ...
+         async def __aenter__(self) -> _T: ...
+         async def __aexit__(self, *exctype: Any) -> None: ...
+-
+-elif sys.version_info >= (3, 7):
```

Comment:
I tried to use `elif True` and `elif False` but because mypy treads `False` as an alias for `typing.TYPE_CHECKING` which is an alias for `True` it gets detected as:

```
contextlib2/__init__.pyi:138: error: Name "nullcontext" already defined on line 126
```